WcaOnRails/config/routes.rb

```diff
@@ -183,6 +183,9 @@
   get '/admin/compute_auxiliary_data' => 'admin#compute_auxiliary_data'
   get '/admin/do_compute_auxiliary_data' => 'admin#do_compute_auxiliary_data'
   get '/admin/update_statistics' => 'admin#update_statistics'
+  get '/admin/reassign_wca_id' => 'admin#reassign_wca_id'
+  get '/admin/validate_reassign_wca_id' => 'admin#validate_reassign_wca_id'
```

Comment:
I think this should still be `post`, because we are sending a form afterall, but I guess that's debatable ^^

Reply:
hmm, maybe. My reasoning behind `get` was a `post` generally creates/modifies data, and with a validate we aren't doing either of those. Let me know if you still want it changed.

Reply:
I see the reasoning and I'm not opinionated here, so if you like `get`, let's go with that!